packages/marko-web-omeda-identity-x/index.js

```diff
@@ -98,6 +101,9 @@ module.exports = (app, params = {}) => {
     omedaPromoCodeCookieName: Joi.string().default('omeda_promo_code'),
     omedaPromoCodeDefault: Joi.string(),
     omedaRapidIdentifyProp: Joi.string().default('$omedaRapidIdentify'),
+    onLoginLinkSentShouldAwait: Joi.boolean().default(false),
```

Comment:
Same here... perhaps make this a re-usable Joi schema and import it into both files